app/src/main/java/com/sdp13epfl2021/projmag/model/Message.kt

```diff
@@ -0,0 +1,5 @@
+package com.sdp13epfl2021.projmag.model
+
+data class Message(val messageContent: String, val sender : ImmutableProfile, val createdAt : Long) {
```

Comment:
This need some KDoc, in particular we can't know from here what createdAt is (timestamp/seconds/miliseconds/etc ?)

Reply:
I agree